packages/utils/src/date-utils/isNewLatest.test.ts

```diff
@@ -1,48 +1,55 @@
 import { MILLIS_PER_HOUR } from './conversionUtils';
 import { isNewLatest } from './isNewLatest';
 
+//TODO: this test should be updated if could be passed by a function always returning true
```

Comment:
hahaha, good idea

how about an event that is contained in the previous? this is very common in our rundowns

```ts
    expect(
      isNewLatest(
        { timeStart: 21 * MILLIS_PER_HOUR, duration: MILLIS_PER_HOUR, dayOffset: 0 },
        { timeStart: 21 * MILLIS_PER_HOUR + 10 * MILLIS_PER_MINUTE, duration: 10 * MILLIS_PER_MINUTE, dayOffset: 0 },
      ),
    ).toBe(false);
```